forem/forem
# Add Mailchimp API Key to Site Config Form

## What type of PR is this? (check all applicable)

## Description
This adds the Mailchimp API key to allow Forems add their own Mailchimp newsletters. This was previously possible, but there just wasn't the form added.

## Related Tickets & Documents
#10270

## QA Instructions, Screenshots, Recordings
1. Go to /admin/config
2. Update the API key in the Newsletter section
3. Check that it persists

## Added tests?

## Added to documentation?
- [x] admin docs
